templates/style.scss

```diff
@@ -114,11 +116,14 @@ div.container-rustdoc {
 }
 
 div.nav-container {
+    // Nothing is supposed to be over or hovering the top navbar. Maybe add a few others '('? :)
+    z-index: 999999999;
```

Comment:
This feels a little outlandish. A z-index of 1 should suffice, but you can also match `div.nav-container-rustdoc` by using 999.